Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -524,10 +582,16 @@ public double MainWindowWidth
 
         public string OpenResultCommandModifiers => Settings.OpenResultModifiers;
 
+        public string TogglePreviewHotkey => Settings.PreviewHotkey; // TODO: is hotkey combo possible?
```

Comment:
Todo still required?

Reply:
Haven't figure out how to use hotkey combo yet. Not going to do it is this pr so let's just keep it.

Reply:
Do we want to create an issue for this instead of a comment?

Reply:
Create a todo is fine. not really a issue because I haven't decide how to change the hotkey and what hotkey can be assigned.

Reply:
In this case I suggest removing the todo, the reason is because we should avoid polluting the code base with todos. If there is something like a tech debt or idea we definitely need to go back to fix/implement we should make an issue to record it. Leaving todos will basically mean rarely anyone will go back to look at it.

Reply:
Plus a todo is only understandable by the person who wrote it, rarely will others know what exactly is involved in a todo so will usually shy away from it. An issue with a proper description will mean someone else will be able to pick it up if it hasnt been by the original author.